changelog/fragments/sdk-base-image.yaml

```
@@ -0,0 +1,34 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      New base image assets containing the Operator-SDK CLI will be built
+      during the release.
```

Nit: this changelog won't show up until the release has been cut, so switch to past tense?

```suggestion
      Added new base image assets containing the Operator SDK CLI at 
      `quay.io/operator-framework/operator-sdk`.
```

---

@joelanford sounds good. I didn't use this exactly, since the base image is built for multiple architectures.